### PR TITLE
Fixes #251 & #418

### LINKED ISSUE #251: Broken command [Remaining] attribute
When command is called with no arguments, command with method signature:
```cs
public void Command(CommandContext context, [Remaining] string args);
```
should work correctly with value of `args` being `""` (an empty string).

Instead, IndexOutOfRangeException is thrown. Cause can be tracked down to somewhere around [here](https://github.com/ObsidianMC/Obsidian/blob/45edc2d06136c84fcb370767a068a410a7033d6a/Obsidian/Commands/Framework/Entities/Command.cs#L114).

### PR DESCRIPTION
**Pull request by TheVeryStarlk**
*(I dunno how I can separate each commit to a different PR)*

### CODE REVIEW

File: Obsidian/Server.cs [hunk 1]
```diff
@@ -331,8 +333,17 @@ private async Task AcceptClientsAsync()
                     // No longer accepting clients.
                     break;
                 }
-
                 connection = acceptedConnection;
+
+                if (!_isFullyInitialized)
```
Comment (Tides) [inline Obsidian/Server.cs]: You can use WorldManager.ReadyToJoin

**Review by Tides — CHANGES_REQUESTED** [review]

**Comment by Naamloos** [comment]
420 blaze it

**Review by Naamloos — CHANGES_REQUESTED** [review]
Apart from tides' requested change this looks fine

File: Obsidian/Server.cs (hunk 1 above)
Comment (TheVeryStarlk) [reply Obsidian/Server.cs]: Should be good now 😄

**Review by Craftplacer — APPROVED** [review]

**Pull request merged**